api-consumption/src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -267,7 +267,6 @@ private void streamToOutput(final boolean streamTimeoutReached) {
                     streamTimeoutReached))) {
                 sentSomething |= !toSend.isEmpty();
                 flushData(e.getKey(), toSend, batchesSent == 0 ? Optional.of("Stream started") : Optional.empty());
-                this.sentEvents += toSend.size();
```

Comment:
In this and in the next block, the `messagesAllowedToSend` is tracked differently.  Should we rather use `getMessagesAllowedToSend()` in both cases?